### DISCUSSION TITLE
using event buttons playback mode

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1116] [Q&A]
What value is by-time ?      Set the result   <2>by-BT ;<3>by-ET.
![20230228162711](https://user-images.githubusercontent.com/116416332/221796396-c62bca85-507e-441d-9006-4b89a3301608.png)

**Comment by MAKOMO** [comment]
This is to configure [event replay](https://artisan-roasterscope.blogspot.com/2017/10/profile-templates.html). Could you please consult the manual and the Artisan blog (it has a search capability) before posting here?

**Reply by Terracotta-62** [reply]
Can you tell me how to set the event button by-time， “sets playback mode to 0: off, 1: time, 2: BT, 3: ET” 。

**Reply by MAKOMO** [reply]
Did you see the button "Help" in the buttons tab of the Events dialog? 

`playbackmode(1)` is by time as you have set your button labeled "playback\nON".

`playbackmode(3)` is by ET.

**Reply by Terracotta-62** [reply]
I misunderstood the logical relation. The default value 1 is "by-time". Thank you for your answer！
